lib/config.js

```diff
@@ -7,3 +7,6 @@ var ENABLE_PUSHSTATE = (IN_STATIC_SITE &&
 exports.IN_STATIC_SITE = IN_STATIC_SITE;
 exports.GENERATING_STATIC_SITE = GENERATING_STATIC_SITE;
 exports.ENABLE_PUSHSTATE = ENABLE_PUSHSTATE;
+
+// this is a dummy account until we choose a domain name
+exports.GA_ACCOUNT='UA-59356678-3';
```

Comment:
I'm dropping offline now, so if you have time to implement today that would be great. Otherwise I can pick up again tomorrow.

Either way, I'm happy to see environment variables in use :+1: